docs/usage/sign_requests.md

```diff
@@ -32,12 +35,23 @@ files = [source_file]
 
 signer = {
     'name': 'John Doe',
-    'email': 'signer@mail.com' 
+    'email': 'signer@mail.com'
 }
 signers = [signer]
 
-parent_folder_id = '123456789'
-new_sign_request = client.create_sign_request(files, signers, parent_folder_id)
+new_sign_request = client.create_sign_request_v2(signers, files)
```

Comment:
Why removing parent_folder_id from docs? I think we should add kwargs to optional names 
`client.create_sign_request(files, signers=signers, parent_folder_id=parent_folder_id)`

Reply:
sure we could keep it, but it's no longer a required arg

Reply:
I know that is why I suggested using kwargs